### DISCUSSION TITLE
Newbie: Connecting over the internet?

### DISCUSSION THREAD

**Discussion by KnoAll** [markqvist/Reticulum#26] [Help & Questions] [answered]
I'm just getting introduced to Reticulum and am wondering if there are any known nodes on the internet that can be connected to? Would be interesting to have a live discussion for the purposes of connecting and validating config, etc.

**Comment by 4c3e** [comment, marked as answer]
@markqvist is kind enough to host a publically available node accessed at the following location:

```
[[RNS Testnet Frankfurt]]
  type = TCPClientInterface
  interface_enabled = yes
  outgoing = True
  target_host = frankfurt.rns.unsigned.io
  target_port = 4965
```
Copy and paste the above declaration into you `.reticulum/config` file (on mac and linux, not sure where this is located on windows). Then download and run [NomadNet](https://github.com/markqvist/nomadnet). Please feel free to comment here if you have any questions, many others likely have the same question, I certainly did when first starting out with Reticulum :smile:.

I've also opened a general where anyone interested in hosting a public RNS node can comment on: https://github.com/markqvist/Reticulum/discussions/27

**Reply by markqvist** [reply]
That's exactly right, thanks for posting this @4c3e!

I will add a section to the repository readme to make this info more visible.

**Reply by markqvist** [reply]
There is also a publicly available node on I2P:

```
  [[RNS Testnet I2P Node A]]
    type = I2PInterface
    interface_enabled = yes
    peers = ykzlw5ujbaqc2xkec4cpvgyxj257wcrmmgkuxqmqcur7cq3w3lha.b32.i2p
```

You will need [i2pd](https://i2pd.website) installed and running to use this.

**Reply by markqvist** [reply]
Courtesy of @4c3e, this information is also [in the manual now](https://markqvist.github.io/Reticulum/manual/gettingstartedfast.html#bridging-over-the-internet).

**Comment by ZS6GJM** [comment]
I have created a Public accessible server, connected to the Testnet server. 
It would be interesting to see if anyone could reach my node at home, connected to my Public server, connected to the Testnet server - Feel free to reach out to me at 98183621d4d3e16a9ab1

**Reply by markqvist** [reply]
A bit overdue here, but is your node still up? Tried messaging you, but couldn't retrieve an announce for your address.

**Reply by ZS6GJM** [reply]
Hi Mark,

The node is still up and running. I restarted my client this morning and I can see the path come through on the public node :-

root@node-0-rns-za-arza-org-za [08:27:57] :~/.reticulum
[2022-04-29 06:28:39] [Debug] Validating announce from <98183621d4d3e16a9ab1>
[2022-04-29 06:28:39] [Debug] Stored valid announce from <98183621d4d3e16a9ab1>
[2022-04-29 06:28:39] [Verbose] Path to <98183621d4d3e16a9ab1> is now 1 hops away via <0bd6a8542702b2e868c5> on TCPInterface[Client on TCP Server Interface/102.132.195.34:59982]
[2022-04-29 06:28:39] [Verbose] Path to <98183621d4d3e16a9ab1> associated with tunnel <6adf9ca2c956610925139326cbb076b3f5196a33b92e3c88ed38700c0abda6e2>

Please try again when you have a moment...

Thanks, Gabriel

**Reply by markqvist** [reply]
Strange, I can't resolve a path to <98183621d4d3e16a9ab1>, neither from my own nodes or the testnet node in Frankfurt. Maybe your public node is not passing on announces. Did you enable transport on it? You're welcome to post the config if you want.

**Reply by ZS6GJM** [reply]
You are right, I thought I'd enabled Transport, but it was set to False. I have enabled it now...

**Reply by 4c3e** [reply]
I was just able to find a path to <98183621d4d3e16a9ab1> on a `rnsd` instance connected to the testnet :smile:

**Reply by ZS6GJM** [reply]
Great, thanks @4c3e for the confirmation on the paths. Could you start a conversation from your side?

**Reply by 4c3e** [reply]
sure I can try now

**Reply by ZS6GJM** [reply]
replied.

**Reply by markqvist** [reply]
I tried sending a couple of messages too. Couldn't deliver them directly when I sent, but they should be waiting for you on the propagation net when your client is back up :)
